netlogo-gui/src/test/headless/TestBehaviorSpace.scala

```diff
@@ -175,19 +175,22 @@ with OneInstancePerTest with BeforeAndAfterEach {
       workspace.report("behaviorspace-experiment-name"))
   }
   // test export-graphics in headless mode
-  if(!Version.is3D)
+  if (!Version.is3D)
     test("ExportGraphics", SlowTest.Tag) {
       val workspace = newWorkspace()
       workspace.open("models/test/lab/FireWithExperiments.nlogo")
       newWorker("testExportGraphics")
         .run(workspace, () => workspace, 1)
     }
-  if(!Version.is3D)
+  if (!Version.is3D)
     test("ModelWithIncludedExperiments", SlowTest.Tag) {
-      runExperimentFromModel("models/test/lab/FireWithExperiments.nlogo", "test1", "models/test/lab/FireWithExperiments1")
-      runExperimentFromModel("models/test/lab/FireWithExperiments.nlogo", "test2", "models/test/lab/FireWithExperiments2")
+      runExperimentFromModel("test/lab/FireWithExperiments.nlogo", "test1", "test/lab/FireWithExperiments1")
+      runExperimentFromModel("test/lab/FireWithExperiments.nlogo", "test2", "test/lab/FireWithExperiments2")
+      runExperimentFromModel("test/lab/FireWithExperiments.nlogo", "test3", "test/lab/FireWithExperiments3")
+      runExperimentFromModel("test/lab/FireWithExperiments.nlogo", "test4", "test/lab/FireWithExperiments4")
+      runExperimentFromModel("test/lab/FireWithExperiments.nlogo", "test5", "test/lab/FireWithExperiments5")
```

Comment:
These changes should also be made to the `netlogo-headless/src/test/headless/misc/TestBehaviorSpace.scala` file as well.  TLDR on that: There was a fork of the NetLogo engine that is being slowly merged back together into `netlogo-core` but many pieces still have two copies, so they need to be kept in sync.